ui/app/AppLayouts/Wallet/HistoryTab.qml

```diff
@@ -205,10 +217,39 @@ Item {
     }
 
     ListView {
+        id: transactionListRoot
         anchors.topMargin: 20
         anchors.fill: parent
+        anchors.bottomMargin: 69
+        anchors.bottom: extraButtons.top
         model: walletModel.transactions
         delegate: transactionListItemCmp
+        ScrollBar.vertical: ScrollBar {
+            id: scrollBar
+        }
+    }
+
+    RowLayout {
+        id: extraButtons
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.bottom: parent.bottom
+        anchors.top: transactionListRoot.bottom
+
+        StatusButton {
+            id: loadMoreButton
+            //% "Load More"
+            text: qsTrId("load-more")
+            anchors.right: parent.right
```

Comment:
You can't use anchors inside a RowLayout (or any Layout). It gives the following error:
```
qrc:/app/AppLayouts/Wallet/HistoryTab.qml:239:9: QML StatusButton: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
```

Instead, you can use ` Layout.alignment: Qt.AlignRight | Qt.AlignVCenter`